src/components/share-url.ts

```diff
@@ -0,0 +1,42 @@
+import { LitElement, css, html } from "lit";
+import { customElement, state } from "lit/decorators.js";
+import { ICON_CHECK_SLOTTED, ICON_COPY_SLOTTED } from "../const";
+
+@customElement("share-url")
+export class ShareURL extends LitElement {
+  @state() private copied: boolean = false;
+
+  render() {
+    return html`
+      <md-text-button
+        trailing-icon
+        @click=${this.copyToClipboard}
+        ?disabled=${this.copied}
```

Comment:
It's also a property, so you can also set it like this:
```suggestion
        .disabled=${this.copied}
```

Reply:
(properties are faster, as you bypass the DOM model and directly talk to the class)